streaming/src/test/scala/org/apache/spark/streaming/ReceivedBlockHandlerSuite.scala

```diff
@@ -62,6 +61,19 @@ class ReceivedBlockHandlerSuite
   var blockManagerMaster: BlockManagerMaster = null
   var blockManager: BlockManager = null
   var tempDirectory: File = null
+  var storageLevel = StorageLevel.MEMORY_ONLY_SER
+
+  private def createBlockManager(
```

Comment:
There are two places where BlockManager instantiation logic is present (10 lines below is the other one). Can you consolidate for cleanliness. That is, make the other one use createBlockManager.